src/Symfony/Component/OptionsResolver/Debug/OptionsResolverIntrospector.php

```diff
@@ -103,6 +103,6 @@ public function getNormalizers(string $option): array
      */
     public function getDeprecationMessage(string $option)
```

Comment:
shouldn't we deprecate this one and replace it with getDeprecation as we did in Config & DI?

Reply:
Done